src/index.js

```diff
@@ -88,6 +89,11 @@ function ParseServer({
   serverURL = requiredParameter('You must provide a serverURL!'),
   maxUploadSize = '20mb'
 }) {
+  
+  // Initialize the node client SDK automatically
+  Parse.initialize(appId, javascriptKey || '', masterKey);
```

Comment:
Nit: The `|| ''` on these two are unnecessary because of the `= || ''` and `requiredParameter('You must provide a serverURL!')` in the argument destructuring.